tests/unittests/suchvim/user-plugin.vader

```diff
@@ -0,0 +1,44 @@
+Before (setUp):
+  source ~/.SUCH-Vim/suchvim/user-plugins.vim
+
+
+Execute (Given user plugin When add plugin Then plugin is added):
```

Comment:
The `Given` part of the test name does not provide any valuable information to understand what this tests does, it is not required here.